File: URI/Lua/macpronalts.lua
n = io.read("*number")
preco = 0
for i=1,n do
    etiqueta = io.read("*number")
    qte = io.read("*number")
    if etiqueta == 1001 then
        preco = preco + qte*1.50
    end
    if etiqueta == 1002 then
        preco = preco + qte*2.50
    end
    if etiqueta == 1003 then
        preco = preco + qte*3.50
    end
    if etiqueta == 1004 then
        preco = preco + qte*4.50
    end
    if etiqueta == 1005 then
        preco = preco + qte*5.50
    end
end
print(string.format("%.2f", preco))

File: URI/Python/MacPronalts.py
n = int(input())
preco = 0
for i in range(n):
    infos = [int(x) for x in input().split()]
    etiqueta = infos[0]
    qte = infos[1]
    if etiqueta == 1001:
        preco = preco + qte*1.50
    if etiqueta == 1002:
        preco = preco + qte*2.50
    if etiqueta == 1003:
        preco = preco + qte*3.50
    if etiqueta == 1004:
        preco = preco + qte*4.50
    if etiqueta == 1005:
        preco = preco + qte*5.50
print("{:.2f}".format(preco))